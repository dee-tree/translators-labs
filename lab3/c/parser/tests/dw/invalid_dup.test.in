dw 1 dup ()