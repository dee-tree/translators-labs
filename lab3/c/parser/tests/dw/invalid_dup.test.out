Token \261 
Expected \256 \262 
